lib/content-services/src/lib/search/services/base-query-builder.service.ts

```diff
@@ -87,6 +93,15 @@ export abstract class BaseQueryBuilderService {
 
     protected userFacetBuckets: { [key: string]: FacetFieldBucket[] } = {};
 
+    get queryFragments(): { [key: string]: any } {
+        return this._queryFragments;
+    }
+
+    set queryFragments(value: { [key: string]: any }) {
```

Comment:
Two things: 
1. Can we set string instead of any because I see that queryFragments field has string as value type (right side)? If it works then it would be better option
2. If no then see following
Any here and in setter below give eslint issue. I know you don't know what the type will be as they can have any type but can we use different type than any like "uknown" or "never"? They are a bit safer alternatives 
https://rahuulmiishra.medium.com/typescripts-any-vs-unknown-vs-never-when-and-how-to-use-them-35d81ea57c01
<img width="1393" height="224" alt="image" src="https://github.com/user-attachments/assets/b27f2e2c-e8e7-4100-bfd3-6118f6d52678" />

Would it have sense to use one of those types?